charts/ccsm-helm/test/integration/integration_test.go

```diff
@@ -0,0 +1,136 @@
+// +build integration
+
+package integration
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"context"
+	"testing"
+
+	"github.com/camunda-cloud/zeebe/clients/go/pkg/zbc"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/grpc"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type integrationTest struct {
+	suite.Suite
+	chartPath   string
+	release     string
+	namespace   string
+	kubeOptions *k8s.KubectlOptions
+}
+
+func TestIntegration(t *testing.T) {
+	t.Parallel()
```

Comment:
My bad, it seems testify doesn't play nicely with parallel tests, see https://github.com/stretchr/testify/issues/187. So we might want to omit this.